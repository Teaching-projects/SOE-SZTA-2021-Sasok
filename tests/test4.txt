100
3
110
4