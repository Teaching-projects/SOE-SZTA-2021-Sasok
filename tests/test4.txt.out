Első egység HP-ja: Első egység DMG-je: Első egység AS-je: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 
A Csata elkezdődött! Harcos és Sámán megtámadták egymást. Életük: Harcos 96, Sámán 107
Sámán megtámadta Harcos, így Harcos élete - 92
Harcos megtámadta Sámán, így Sámán élete - 104
Sámán megtámadta Harcos, így Harcos élete - 88
Sámán megtámadta Harcos, így Harcos élete - 84
Harcos megtámadta Sámán, így Sámán élete - 101
Sámán megtámadta Harcos, így Harcos élete - 80
Harcos megtámadta Sámán, így Sámán élete - 98
Sámán megtámadta Harcos, így Harcos élete - 76
Sámán megtámadta Harcos, így Harcos élete - 72
Harcos megtámadta Sámán, így Sámán élete - 95
Sámán megtámadta Harcos, így Harcos élete - 68
Sámán megtámadta Harcos, így Harcos élete - 64
Harcos megtámadta Sámán, így Sámán élete - 92
Sámán megtámadta Harcos, így Harcos élete - 60
Harcos megtámadta Sámán, így Sámán élete - 89
Sámán megtámadta Harcos, így Harcos élete - 56
Sámán megtámadta Harcos, így Harcos élete - 52
Harcos megtámadta Sámán, így Sámán élete - 86
Sámán megtámadta Harcos, így Harcos élete - 48
Harcos megtámadta Sámán, így Sámán élete - 83
Sámán megtámadta Harcos, így Harcos élete - 44
Sámán megtámadta Harcos, így Harcos élete - 40
Harcos megtámadta Sámán, így Sámán élete - 80
Sámán megtámadta Harcos, így Harcos élete - 36
Sámán megtámadta Harcos, így Harcos élete - 32
Harcos megtámadta Sámán, így Sámán élete - 77
Sámán megtámadta Harcos, így Harcos élete - 28
Harcos megtámadta Sámán, így Sámán élete - 74
Sámán megtámadta Harcos, így Harcos élete - 24
Sámán megtámadta Harcos, így Harcos élete - 20
Harcos megtámadta Sámán, így Sámán élete - 71
Sámán megtámadta Harcos, így Harcos élete - 16
Harcos megtámadta Sámán, így Sámán élete - 68
Sámán megtámadta Harcos, így Harcos élete - 12
Sámán megtámadta Harcos, így Harcos élete - 8
Harcos megtámadta Sámán, így Sámán élete - 65
Sámán megtámadta Harcos, így Harcos élete - 4
Sámán megtámadta Harcos, így Harcos élete - 0

SáMáN GYOZEDELMESKEDETT!
